src/plugins/data/public/ui/filter_bar/filter_bar.tsx

```diff
@@ -119,66 +118,65 @@ function FilterBarUI(props: Props) {
   }
 
   function onAdd(filter: Filter) {
+    reportUiCounter?.(METRIC_TYPE.CLICK, `filter:added`);
     setIsAddFilterPopoverOpen(false);
-    if (props.trackUiMetric) {
-      props.trackUiMetric(METRIC_TYPE.CLICK, `${props.appName}:filter_added`);
-    }
+
     const filters = [...props.filters, filter];
     onFiltersUpdated(filters);
   }
 
   function onRemove(i: number) {
+    reportUiCounter?.(METRIC_TYPE.CLICK, `filter:removed`);
     const filters = [...props.filters];
     filters.splice(i, 1);
     onFiltersUpdated(filters);
   }
 
   function onUpdate(i: number, filter: Filter) {
-    if (props.trackUiMetric) {
-      props.trackUiMetric(METRIC_TYPE.CLICK, `${props.appName}:filter_edited`);
-    }
+    reportUiCounter?.(METRIC_TYPE.CLICK, `filte:edited`);
```

Comment:
Reading `usageCollector` README:

> The function
accepts three arguments, `AppName`, `metricType` and `eventNames`. **These should be underscore-delimited strings.**


I think it is a technical limitation and we must use `underscore-delimited` strings for eventNames.

Reply:
I confirmed with the telemetry team this is ok